Resolves #1796 (#2217)
MicrosoftDocs/windows-powershell-docs: docset/windows/storage/New-Partition.md

## EXAMPLES

### Example 1: Create a new partition on disk 1
```powershell
PS C:\> New-Partition -DiskNumber 1 -UseMaximumSize -DriveLetter T
```

> This example creates a new partition on disk 1, using the maximum available space, and assign a drive letter T.

Reviewer: ```suggestion
This example creates a new partition on disk 1 using the maximum available space, and assigns a drive letter T.
```

### Example 2: Get all RAW disks, initialize the disks, partition, and format them
```powershell
PS C:\> Get-Disk | Where-Object PartitionStyle -Eq "RAW" | Initialize-Disk -PassThru | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume
```

> This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then format them.

Reviewer: ```suggestion
This example uses five cmdlets and the pipeline to get all disks, filters them for only RAW, unpartitioned disks, initializes the disks, partitions the disks, and then formats them.
```

> This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then format them.

Reviewer: ```suggestion
This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then to format them.
```

> ### Example 3: Create a new EFI partition on on GTP disk 2

Reviewer 2: ```suggestion
### Example 3: Create a new EFI partition on GTP disk 2
```

```powershell
PS C:\> New-Partition -DiskNumber 2 -Size 500MB -GptType "{c12a7328-f81f-11d2-ba4b-00a0c93ec93b}"
```

> This example creates a new EFI partition on disk 2 with size 500MB.

Author: ```suggestion
This example creates a new EFI partition on disk 2 with size 500 MB.
```

> This example creates a new EFI partition on disk 2 with size 500 MB.

Reviewer: ```suggestion
This example creates a new EFI partition on disk 2 with a size of 500 MB.
```

Reviewer: Apologies @velkovb I seem to have missed this one.

Author: Ah, thanks for spotting that. Seems like I did half of it :)

### Example 4: Create a Windows/system partition on MBR disk 0
```powershell
PS C:\> New-Partition -DiskNumber 0 -Size 100GB -MbrType IFS -IsActive
```

> This example creates a new Windows/system partition on MBR disk 0 with size 100GB.

Reviewer: ```suggestion
This example creates a new Windows/system partition on MBR disk 0 with a size of 100 GB.
```
